python/src/UniformOverMesh_doc.i.in

```diff
@@ -0,0 +1,77 @@
+%feature("docstring") OT::UniformOverMesh
+"UniformOverMesh distribution.
+
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
+Parameters
+----------
+mesh : :class:`~openturns.Mesh`
+    Mesh defining the domain of the distribution.
+
+Notes
+-----
+Its probability density function is defined as:
+
+.. math::
+
+    f_X(x) = \frac{1}{|\cD|}, \quad x \in \cD
+
+with :math:`\cD\subset\Rset^n` a mesh of dimension :math:`n`.
+
+Examples
+--------
+Create a distribution:
+
+>>> import openturns as ot
+>>> import openturns.experimental as otexp
+>>> f = ot.SymbolicFunction(['x', 'y'], ['sin(x)*sin(y)'])
+>>> levelSet = ot.LevelSet(f, ot.Less(), 0.2)
+>>> box = ot.Interval([-5.0]*2, [5.0]*2)
+>>> mesh = ot.LevelSetMesher([50]*2).build(levelSet, box, False)
+>>> distribution = otexp.UniformOverMesh(mesh)
+
+Draw a sample:
+
+>>> sample = distribution.getSample(5)"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::UniformOverMesh::getMesh
+"Accessor to the distribution's mesh.
+
+Returns
+-------
+mesh : ;class:`~openturns.Mesh`
+    Mesh."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::UniformOverMesh::getIntegrationAlgorithm
+"Accessor to the integration algorithm used to compute the CDF.
+
+Returns
+-------
+algo : :class:`~openturns.IntegrationAlgorithm`
+    Integration algorithm used to compute the CDF. Default value is "
```

Comment:
The default value is?

Reply:
Suspens... `IteratedQuadrature` ;-)! From time to time I am called by someone right in the middle of a sentence...